src/js/profile/mobile/widget/Calendar.js

```diff
@@ -269,45 +281,129 @@
 							!targetClassList.contains(classes.SELECTION) &&
 							!targetClassList.contains(classes.DISABLED)) {
 							if (targetClassList.contains(classes.PREV_MONTH_DAY)) {
-								self._moveMonth("left");
+								self._moveMonth(-1);
```

Comment:
Is there any special reason for changing from string to number?

Reply:
Earlier, you had to do a direction condition in method `_moveMonth`. There is no need now. We just add + 1 / -1.
Code below is not optimal because it has too much redundant. This needs to be improved during these two widgets will be combined.